packages/config/src/layer2s/templates/polygonCDKStack.ts

```diff
@@ -0,0 +1,548 @@
+                  {
+                    type: 'liveness',
+                    subtype: 'stateUpdates',
+                  },
+                  {
+                    type: 'l2costs',
+                    subtype: 'stateUpdates',
+                  },
+                ],
+                query: {
+                  formula: 'functionCall',
+                  address: EthereumAddress(
+                    '0x5132A183E9F3CB7C848b0AAC5Ae0c4f0491B7aB2',
+                  ),
+                  selector: '0x1489ed10',
+                  functionSignature:
+                    'function verifyBatchesTrustedAggregator(uint32,uint64,uint64,uint64,bytes32,bytes32,address,bytes32[24])',
+                  sinceTimestampInclusive: new UnixTime(1707822059),
+                },
+              },
+              {
+                uses: [
+                  {
+                    type: 'liveness',
+                    subtype: 'stateUpdates',
+                  },
+                  {
+                    type: 'l2costs',
+                    subtype: 'stateUpdates',
+                  },
+                ],
+                query: {
+                  formula: 'functionCall',
+                  address: EthereumAddress(
+                    '0x5132A183E9F3CB7C848b0AAC5Ae0c4f0491B7aB2',
+                  ),
+                  selector: '0x87c20c01',
+                  functionSignature:
+                    'function verifyBatches(uint32,uint64,uint64,uint64,bytes32,bytes32,address,bytes32[24])',
+                  sinceTimestampInclusive: new UnixTime(1707822059),
+                },
+              },
+            ],
+      liveness: {
+        duplicateData: {
+          from: 'stateUpdates',
+          to: 'proofSubmissions',
+        },
+      },
+      finality:
+        templateVars.daProvider !== undefined ? undefined : 'coming soon',
+    },
+    chainConfig: templateVars.chainConfig,
+    dataAvailability:
+      daProvider !== undefined
+        ? addSentimentToDataAvailability({
+            layers: daProvider.fallback
+              ? [daProvider.name, daProvider.fallback]
+              : [daProvider.name],
+            bridge: daProvider.bridge,
+            mode: 'Transactions data',
+          })
+        : addSentimentToDataAvailability({
+            layers: ['Ethereum (calldata)'],
+            bridge: { type: 'Enshrined' },
+            mode: 'Transactions data',
+          }),
+    riskView: makeBridgeCompatible({
+      stateValidation: {
+        ...RISK_VIEW.STATE_ZKP_SN,
+        sources: [
+          {
+            contract: 'PolygonRollupManager',
+            references: [
+              'https://etherscan.io/address/0x5132A183E9F3CB7C848b0AAC5Ae0c4f0491B7aB2',
+            ],
+          },
+        ],
+      },
+      dataAvailability: {
+        ...riskViewDA(daProvider),
+        description:
+          riskViewDA(daProvider).description +
+          ' Unlike most ZK rollups transactions are posted instead of state diffs.',
```

Comment:
To remove as it's not true for all cdk chains, i.e. validiums post only hash of transactions